src/MockBot.Api/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```

Comment:
How come the non-Development settings are the ones that allow all hosts? My instinct would be for it to be the other way round.